os/net/Kconfig

```diff
@@ -119,6 +119,7 @@ source "$EXTERNALDIR/tftpc/Kconfig.protocol"
 source "$EXTERNALDIR/telnetd/Kconfig.protocol"
 source "$EXTERNALDIR/smtp/Kconfig.protocol"
 
+source "$EXTERNALDIR/slsi_wifi/Kconfig"
```

Comment:
Please, remove this.
I will add:
menu "Wireless"

config WIFI_MANAGER
    bool "Enable Wi-Fi Manager"
    default n
    ---help---
        Easy APIs for applications to use and control Wi-Fi features

if WIFI_MANAGER
source "$EXTERNALDIR/slsi_wifi/Kconfig"
endif #WIFI_MANAGER

endmenu #Wireless

Reply:
It's temporal modification. I can't test TLS now without above config because there are no control programs for wifi in TizenRT. I think it would be better to fix it when you complete to add wifi manager